CHANGELOG.md

```diff
@@ -606,4 +606,4 @@ version of all the changes since `v0.9.1`.
 - Switched from `nose` to `pytest` for unit tests.
   - Unit tests take 1 minute now instead of 8
 - Massively expanded documentation
-- Docs are now hosted on [spack.readthedocs.io](http://spack.readthedocs.io)
+- Docs are now hosted on [spack.readthedocs.io](https://spack.readthedocs.io)
```

Comment:
I think these https changes got sucked into this PR. I would remove them to keep the PR simple.

Reply:
If you want me to remove them, please let me know if you should reject this pull request and I should start a new one where only my aocc_compiler changes are present.